.github/workflows/e2e-tests.yml

```diff
@@ -18,17 +18,31 @@ on:
       - "cypress/**"
 jobs:
   smoke_tests:
-    runs-on: ubuntu-latest
+    container:
+      image: cypress/browsers:node-20.14.0-chrome-126.0.6478.114-1-ff-127.0.1-edge-126.0.2592.61-1
+      options: --user root
+    runs-on: self-hosted
```

Comment:
**note:** For future reference, once [IOXTD-2520](https://jira.corp.adobe.com/browse/IOXTD-2520) (more specifically [IOXTD-2789](https://jira.corp.adobe.com/browse/IOXTD-2789)) is completed, we'll have self-hosted runners on Ethos Corp-Connected Clusters so that we can replace use of this "temporary" self-hosted runner (see [IOXTD-2788](https://jira.corp.adobe.com/browse/IOXTD-2788)).